ethexe/service/src/lib.rs

```diff
@@ -235,6 +229,7 @@ impl Service {
 
         let fast_sync = config.node.fast_sync;
 
+        #[allow(unreachable_code)]
```

Comment:
directly on use place

Reply:
`unreachable!()` applies to the whole expression (e.g. `Ok(...)`), so it's impossible to move it elsewhere